scripts/Compile-Zstddec.ps1

```diff
@@ -0,0 +1,14 @@
+# This PowerShell script (principally for Windows 10/11) compiles zstddec.js using docker according to the values set in 
+# /emscripten/ztandard/compile.sh . Please be sure to review the commandline in compile.sh before running this script, to
+# be sure it's doing what you want.
+#
+# Prerequisites:
+#
+#     * Ensure WSL 2 is enabled in Windows (WSL = Windows Subsystem for Linux) 
+#     * Install docker desktop on Windows: choose the WSL 2 docker backend (not the standard/legacy one)
+#     * Open docker and ensure the docker daemon is running correctly (icon should be green)
+#     * Ensure docker desktop is signed in to your dockerhub account
+#     * Run this script by opening a PowerShell terminal, cd to the kiwix-js repo and run './scripts/Compile-Zstddec.ps1'
```

Comment:
If I understand correctly your line below, you properly get the directory of the repo, so I suppose your script could be ran from any current directory?
That's what I'm going to do in bash

Reply:
Yes, `$PSScriptRoot` always resolves to the full path of the directory the running script is in.